trollmoves/tests/test_movers.py

```diff
@@ -88,6 +89,30 @@ def test_s3_copy_file_to_base(S3FileSystem):
     S3FileSystem.return_value.put.assert_called_once_with(ORIGIN, "data-bucket/filename.ext")
 
 
+@patch('trollmoves.movers.S3FileSystem')
+def test_s3_get_destination_from_string(S3FileSystem):
+    """Test get the correct S3-bucket destination filename."""
+    s3_mover = _get_s3_mover(ORIGIN, "s3://data-bucket/")
+    result = s3_mover._get_destination()
+    assert result == 'data-bucket/filename.ext'
```

Comment:
Do not use the private method (and it should stay private, IMO). Instead check that the mocked `S3FileSystem.return_value.put()` has been called with the expected arguments.